colorbleed/plugins/maya/create/colorbleed_pointcache.py

```diff
@@ -15,22 +13,14 @@ class CreatePointCache(avalon.maya.Creator):
     def __init__(self, *args, **kwargs):
         super(CreatePointCache, self).__init__(*args, **kwargs)
 
-        # create an ordered dict with the existing data first
-        data = OrderedDict(**self.data)
+        data = {"writeColorSets": False,  # Vertex colors with the geometry.
+                "renderableOnly": False,  # Only renderable visible shapes
+                "visibleOnly": False,  # only nodes that are visible
+                "attr": "cbId",  # Add options for custom attributes
```

Comment:
Since `cbId` is crucial to the pipeline I would force it in [here](https://github.com/Colorbleed/colorbleed-config/pull/192/files#r225471259) and remove it from needing to be included in the `attr` data on the instance. :)

What do you think @aardschok ?